src/volume.ts

```diff
@@ -220,3 +245,44 @@ export enum EmptyDirMedium {
    */
   MEMORY = 'Memory'
 }
+
+export interface SecretVolumeOptions {
```

Comment:
Please add a docstring to this interface.

Reply:
Done